### ISSUE TITLE
TCN model code error

### ISSUE THREAD

**Issue by yycyyyc321** [microsoft/qlib#1401]
## 🐛 Bug Description

qlib/contrib/model/pytorch_tcn_ts.py
line 171-line174
            feature = data[:, :, 0:-1].to(self.device)
            label = data[:, -1, -1].to(self.device)

            pred = self.TCN_model(feature.float())

## To Reproduce

1.  change seq len as 10, and run \\examples\\benchmarks\\TCN\\workflow_config_tcn_Alpha158.yaml
1.
1.

## Screenshot

![image](https://user-images.githubusercontent.com/20674896/209767568-094c8a4d-3034-492a-a15c-2452f39c91b4.png)
x as input with shape(batch,seq_len,feat), but right input should be as shape(batch,feat,seq_len,) so nn.Conv1d can be calculate
## Environment

**Note**: User could run `cd scripts && python collect_info.py all` under project directory to get system information
and paste them here directly.

 - Qlib version:
 - Python version:
 - OS (`Windows`, `Linux`, `MacOS`):
 - Commit number (optional, please provide it if you are using the dev version):

**Comment by chuhuan88** [comment]
Hello, can you share the downloaded dataset with me, I can't get on Yahoo, thank you very much.
